### PR TITLE
Update research.rst

### PR DESCRIPTION
**Pull request by DecoderLiu**
Add university of victoria and university of british columbia to the list.

### CODE REVIEW

**Comment by lululxvi** [comment]
You missed the afflictions in the economics PINN paper.

**Comment by DecoderLiu** [comment]
> You missed the afflictions in the economics PINN paper.

I think all affiliations in that paper are already included(Princeton, MIT, USC, UPenn)

**Comment by lululxvi** [comment]
I don't see Princeton

**Comment by lululxvi** [comment]
Also, order the universities by US News global ranking.

File: docs/user/research.rst [hunk 1]
```diff
@@ -9,10 +9,12 @@ DeepXDE has been used in
   `University of California, Berkeley <https://www.berkeley.edu>`_,
   `Johns Hopkins University <https://www.jhu.edu>`_,
   `University of Pennsylvania <https://www.upenn.edu>`_,
+  `Princeton University <https://www.princeton.edu/>`_,
```
Comment (lululxvi) [inline docs/user/research.rst:12]: Delete the last `/` in the url

**Pull request merged**